docs/examples/elephantsdream/index.html

```diff
@@ -24,15 +23,15 @@
     <source src="//d2zihajmogu5jn.cloudfront.net/elephantsdream/ed_hd.mp4" type="video/mp4">
     <source src="//d2zihajmogu5jn.cloudfront.net/elephantsdream/ed_hd.ogg" type="video/ogg">
 
-    <track kind="captions" src="captions.en.vtt" srclang="en" label="English" default></track><!-- Tracks need an ending tag thanks to IE9 -->
-    <track kind="captions" src="captions.sv.vtt" srclang="sv" label="Swedish"></track>
-    <track kind="captions" src="captions.ru.vtt" srclang="ru" label="Russian"></track>
-    <track kind="captions" src="captions.ja.vtt" srclang="ja" label="Japanese"></track>
-    <track kind="captions" src="captions.ar.vtt" srclang="ar" label="Arabic"></track>
+    <track kind="captions" src="captions.en.vtt" srclang="en" label="English" default>
```

Comment:
we technically still need these since 6.x supports ie9...

Reply:
Oh, whoops! Didn't mean for any of those changes to be in this PR!